tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -1095,7 +1097,7 @@ def create_and_validate_custom_host(node_roles, random_cluster_name=False,
 
 def create_custom_host_from_nodes(nodes, node_roles,
                                   random_cluster_name=False, windows=False,
-                                  version=K8S_VERSION):
+                                  version=K8S_VERSION, rke_config=rke_config):
```

Comment:
We are including rke_config in the list of variables for the method, how did it work before?
Are there other calls to this method which will need modification ?